packages/ragbits-document-search/src/ragbits/document_search/_main.py

```diff
@@ -128,7 +128,44 @@ async def ingest_document(
         if document_processor is None:
             document_processor = self.document_processor_router.get_provider(document_meta)
 
-        elements = await document_processor.process(document_meta)
+        document_processor = self.document_processor_router.get_provider(document_meta)
+        return await document_processor.process(document_meta)
+
+    async def ingest_document(
```

Comment:
I would probably want to remove `ingest_document` all together. And then maybe even rename `ingest_documents` to simply `ingest`. What do you think?

Reply:
I like the idea of a single endpoint + `ingest` is concise, I like it.